src/components/text-field.js

```diff
@@ -41,8 +41,8 @@ const _inputSize = ({ size, variant }) => {
 
 const _textFieldClasses = ({ container, inputSize, success, error, linkedFields }) => ({
   container: {
-    default: `flex flex-col font-sans ${container ? container : 'w-250px'} ${inputSize.font}`,
-    borderless: `${container ? container : 'w-250px'} bg-secondary-200`,
+    default: `textfield__container ${container ? container : 'w-250px'} flex flex-col font-sans ${inputSize.font}`,
+    borderless: `textfield__container ${container ? container : 'w-250px'} bg-secondary-200`,
```

Comment:
Can we call these `textfield__default-container` and `textfield__borderless-container` so it's easier to read later on